packages/core/src/components/carousel/carousel.tsx

```diff
@@ -66,6 +66,18 @@ export class AtomCarousel {
 
   componentDidLoad() {
     this.swiperEl = this.host.querySelector('swiper-container')
+
+    const children = this.host.querySelectorAll('atom-carousel-item')
+
+    children.forEach((child) => {
```

Comment:
(question) we need to change something in documentation of this component?

Reply:
Actually No! The component is used exactly in the same way. Just the strategy to normalize atom-carousel-item is diferent